event_manager/lib/event_manager.rb

```diff
@@ -7,25 +7,49 @@
 
 puts "EventManager initialized."
 
-contents = CSV.open 'event_attendees.csv', headers: true, header_converters: :symbol
-
-template_letter = File.read "form_letter.erb"
-erb_template = ERB.new template_letter
-
-contents.each do |row|
-  id = row[0]
-  name = row[:first_name]
-  zipcode = clean_zipcode(row[:zipcode])
-  legislators = legislators_by_zipcode(zipcode)
-  form_letter = erb_template.result(binding)
-  phone = sanitize_phone(row[:homephone])
-  phone = validate_phone(phone)
-  date = parse_date(row[:regdate])
-  hour =  date.hour
-  minutes = date.min
-  week_day = date.wday
- 
-  save_thank_you_letters(id,form_letter)
+def get_data
```

Comment:
this method should be called something more like `generate_confirmation_letters`